public/app/types/unified-alerting-dto.ts

```diff
@@ -258,18 +263,19 @@ export interface PostableGrafanaRuleDefinition {
     metric: string;
     from: string;
   };
+  intervalSeconds?: number;
```

Comment:
Our types are a bit confusing. This wasn't defined but it is a property on the API response? Am I missing something? For now, added it as optional but the types need unravelling for sure